app/assets/javascripts/Rickshaw.Graph.DragZoom.js

```diff
@@ -0,0 +1,131 @@
+Rickshaw.namespace('Rickshaw.Graph.DragZoom');
```

Comment:
@stuartnelson3 What is the state on contributing this back to rickshaw? Still planned?